src/extractors/youtube-new/utils/constants.ts

```diff
@@ -0,0 +1,16 @@
+export default {
+  URL: {
+    YT_URL: "https://www.youtube.com",
+    YT_MOBILE: "https://m.youtube.com",
+    YT_MUSIC_URL: "https://music.youtube.com",
+    YT_BASE_API: "https://www.youtube.com/youtubei/v1",
+    YT_SUGGESTION_API: "https://suggestqueries.google.com/complete/search",
+  },
+  YTAPIVAL: {
+    VERSION: "17.20",
+    CLIENTNAME: "ANDROID",
+    VERSION_WEB: "2.20220411.09.00",
```

Comment:
Upgrade to Version 2.20230206.06.00 just so that we have the latest features when developing

Reply:
updated in fd1001e